src/ansys/mechanical/core/run.py

```diff
@@ -102,6 +103,13 @@ def _run(args, env, check=False, display=False):
     type=int,
     help="Start mechanical in server mode with the given port number",
 )
+@click.option(
+    "--features",
+    type=str,
+    default=None,
```

Comment:
https://mechanical.docs.pyansys.com/version/stable/getting_started/running_mechanical.html , docs need update

Reply:
@dipinknair thanks. I think we should split out a separate section for running mechanical using ansys-mechanical. The title of that document is "launching Pymechanical" which isn't exactly right.

Reply:
Yes. I also like the idea of having separate section for `ansys-mechanical`

Reply:
@dipinknair I filed an issue for having a separate doc section